src/components/QuranReader/ReadingPreferenceSwitcher.tsx

```diff
@@ -30,15 +33,28 @@ const ReadingPreferenceSwitcher = () => {
     ],
     [t],
   );
+
+  const onViewSwitched = (view: ReadingPreference) => {
+    logValueChange('reading_preference', readingPreference, view);
+
+    // drop `startingVerse` from query params
+    const newQueryParams = omit(router.query, ['startingVerse']);
```

Comment:
I think we don't need to use `omit` at all. You can check a reference for what you are trying to do inside `src/components/Navbar/SettingsDrawer/ResetButton.tsx`.`onResetSettingsClicked`

Reply:
got it 👍